src/ussd_tb_check.js

```diff
@@ -660,7 +660,12 @@ go.app = function () {
       return new JsonApi(self.im)
         .post(self.im.config.healthcheck.url + "/v2/tbcheck/", payload)
         .then(
-          function () {
+          function (response) {
+            self.im.user.answers = {
```

Comment:
Won't his overwrite the whole answers object with a new one? I think it's better to set each value separately

Reply:
Oh yes it does override it thanks.